packages/lit-dev-content/site/docs/components/lifecycle.md

```diff
@@ -381,18 +381,24 @@ updated(changedProperties) {
 
 The `updateComplete` Promise resolves when the element has finished updating. Use `updateComplete` to wait for an update. The resolved value is a Boolean indicating if the element has finished updating. It will be `true` if there are no pending updates after the update cycle has finished.
```

Comment:
```suggestion
The `updateComplete` promise resolves when the element has finished updating. Use `updateComplete` to wait for an update. The resolved value is a boolean indicating if the element has finished updating. It will be `true` if there are no pending updates after the update cycle has finished.
```